src/arkode/arkode_splittingstep.c

```diff
@@ -253,20 +237,50 @@ static int splittingStep_SequentialMethod(ARKodeMem ark_mem,
        * FSAL property). Care is needed when a reset occurs, the step direction
        * changes, the coefficients change, etc. */
       SUNErrCode err = SUNStepper_Reset(stepper, t_start, y);
-      if (err != SUN_SUCCESS) { return ARK_SUNSTEPPER_ERR; }
+      if (err != SUN_SUCCESS)
+      {
+        SUNLogInfo(ARK_LOGGER, "end-partition",
+                   "status = failed stepper reset, err = %i", err);
+        SUNLogInfo(ARK_LOGGER, "end-stage",
+                   "status = failed partition, err = %i", err);
+        return ARK_SUNSTEPPER_ERR;
+      }
 
       err = SUNStepper_SetStepDirection(stepper, t_end - t_start);
-      if (err != SUN_SUCCESS) { return ARK_SUNSTEPPER_ERR; }
+      if (err != SUN_SUCCESS)
+      {
+        SUNLogInfo(ARK_LOGGER, "end-partition",
+                   "status = failed set direction, err = %i", err);
+        SUNLogInfo(ARK_LOGGER, "end-stage",
+                   "status = failed partition, err = %i", err);
+        return ARK_SUNSTEPPER_ERR;
+      }
 
       err = SUNStepper_SetStopTime(stepper, t_end);
-      if (err != SUN_SUCCESS) { return ARK_SUNSTEPPER_ERR; }
+      if (err != SUN_SUCCESS)
+      {
+        SUNLogInfo(ARK_LOGGER, "end-partition",
+                   "status = failed set stop time, err = %i", err);
+        SUNLogInfo(ARK_LOGGER, "end-stage",
+                   "status = failed partition, err = %i", err);
+        return ARK_SUNSTEPPER_ERR;
+      }
 
       sunrealtype tret = ZERO;
       err              = SUNStepper_Evolve(stepper, t_end, y, &tret);
```

Comment:
Perhaps we can save this for a future PR, but I'd think that it would be good to print `y` here when running with "extra debugging" enabled.